rules.txt

```diff
@@ -0,0 +1,148 @@
+
+## Mapa ##
+- Planetas variados dividos por setor tentando seguir o mapa oficial de SW
+- Rotas poderia ser esse esquema dos Hexágonos mesmo ou rotas pré definidas, como é hyperespaço uma CS não necessariamente tem q seguir a rota mas quando der um jump de 2 planetas ele contabilizada o tempo de viagem pela rota. A frota fica em "suspensão"
+- Mostrar os recursos do planeta, a % de dominação e quanto ele está produzindo para o squad
+- Alguns recursos podem ser extraidos com unidades especificas tipo Asteroid Mining Unit e outros por diplomacia, comércio etc.
+- Mostra o "fog of war", as ultimas unidades q ele viu naquele planeta ja descontadas as perdas do combate
+- Não ser possivel ver unidades q estão dentro de CS, barracks ou shipyards. Exceto se espião for bem sucedido.
+
+## Turnos ##
+- Esquadroes fazem suas produções e ordens de movimento na fase de estratégia
+- Segunda fase os movimentos são aplicados e caso haja combates possibilita escolher taticas e aplicar os resultados
+- Terceira fase aplica os resultados e desembarca tropas para uma rodada de batalhas terrestres, regras similares a espacial
+- Quarta fase aplica os resultados, calcula os incomes, atualiza os planetas etc
+- Quanto mais informação tiver do que aconteceu melhor
+
+## Dominação ##
+- A dominação total se daria após 1 ou 2 turnos inteiros com presença única do esquadrão em uma(ou mais) frentes: militar espacial, terrestre ou diplomatica. Com isso habilitaria o income total e a construção de fábricas, mineradoras e barracks. Se 2 ou mais esquadrões estiverem dominando o planeta o income é dividido pela porcentagem de dominação mas sem 
+possibilidade de construção
+
+## Income ##
+- Cada planeta teria uma qtde x de income de creditos, metais e raros dado ao esquadrao na porcentagem de dominação do planeta
+
+## Dados do BD que consigo imaginar até agora##
+Fiz um brainstorm de ideias pro BD. Muita coisa dá pra agrupar em métodos simples. Por exemplo: Uma CS tem load_capacity de 1000. Ela conseguiria carregar tudo que tivesse load_weigth abaixo disso. Um caça teria load_capacity de 1, logo poderia carregar 1 pack de missil dando a ela a role e bonus de ataque correspondente.
+CS tem hiperdrive 3 q permite dar jumps distantes. Caça peso 10 e hiperdrive 1. Troopers peso 1, Facilities peso 2000, ambos hiperdrive 0. Daí vc ja consegue colocar as naves
+de transporte correspondentes sem usar booleans.
+
+## Units ##
+type: String
+terrain: String (ground, space etc)
+name: String
+acronym: String
+credits_cost: Integer
+metals_cost: Integer
+rare_elements_cost: Integer
+producing_points: Integer
+load_weight: Integer
+load_capacity: Integer
+hyperdrive: Integer
+production_time: Integer
+visible: Boolean
+groupable: Boolean
+weaponable: Boolean
+upgradable: Boolean
+character: Boolean
+shield: Integer
+hull: Integer
+initiative: Integer
+attack: Integer
+defense: Integer
+agility: Integer
+diplomacia: Integer
```

Comment:
English please, mas dopraria no primeiro momento. Tambem acho que unidades tao diferentes nao deveriam usar a mesma classe aqui, mas podemos conversar melhor depois que for reintroduzir essas coisas

Reply:
Sure! Essas sao as caracteristicas dos herois q o Dimy vai mandar, coloquei assim pq nao manjo. Sobre as classes é exatamente o q espero q definamos logo rs.